### ISSUE TITLE
[Documentation Update]: JMRI sensors definitions in the CS

### ISSUE THREAD

**Issue by Condo-Colin** [DCC-EX/dcc-ex.github.io#488]
### Documentation details

Discord message from KC re JMRI sensors requiring config in mySetup.h vs EX-RAIL doesn't (RAM usage discussion).

https://discord.com/channels/713189617066836079/713190771125387295/1039956806195228762

Note: documentation place holder

Commenting out the six sensors and the Serial2.begin in my setup allows for the CS to Operate in a Station STA Mode with Engine Driver Direct connect mode and for the Automation(n) Route(n) and Devices to operate with a single ED Throttle and two Locos.

The Sensors no longer display in the Serial monitor Nor do the show up in JMRI Sensor Pane or activate their status in on the screen or in Traffic Monitor,  As Expected.   

So this will need to be documented that a balance will need to be made between what you want to do if your DCC-EX CS is standalone WiFi operations verses if you want it operating thru JMRI and use Sensor and Output panes.

In other words EXRAIL does not require sensors to be previously created in mySetup.h to operate, JMRI does need Sensors to be previously created in either mySetup.h or through DCC++ Config Base Station pane on the main menu.

**Comment by peteGSX** [comment]
Update on this topic from @FrightRisk clarifying how we might be able to offer some guidance on how users can determine how many objects could be defined before running into RAM issues:

I think we need to go ahead with reducing things like max locos and max consists, etc. None of that needs to be that large for 96% of our users. And since we can include a constant that can be changed, people can always bump it up.
And this is what I was talking about earlier with regard to having a table of memory usage. We need a document that shows the cost of different things to remind people, for example, that defining turnouts costs 10 bytes. Then we can have examples showing "10 of these and 5 of these and an EX-RAIL script this long, boom." and one or two more example setups.

We will need some technical detailed input from @Asbelos and possibly @habazut to be able to provide this level of guidance though.
